nomad/datasource_nomad_job.go

```diff
@@ -242,6 +242,10 @@ func dataSourceJob() *schema.Resource {
 										Type:     schema.TypeBool,
 										Computed: true,
 									},
+									"auto_promote": {
+										Type:     schema.TypeBool,
+										Computed: true,
+									},
```

Comment:
please add a test to `datasource_nomad_job_test.go` to test this is working; i don't believe that it is.